synapse/handlers/relations.py

```diff
@@ -123,10 +117,10 @@ async def get_relations(
             room_id=room_id,
             relation_type=relation_type,
             event_type=event_type,
-            limit=limit,
-            direction=direction,
-            from_token=from_token,
-            to_token=to_token,
+            limit=pagin_config.limit,
```

Comment:
mypy unhappy here. Is this just an `assert expr is not None`? (Having said that, I'm wondering if PaginationConfig.limit being None makes sense?)

Reply:
Although it looks invasive enough that I might go with the assert for now?

Reply:
Naively I'd expect "don't use a limit" to be represented by a lack of PaginationConfig.

Happy with an assert for now!